lib/opentelemetry-bridge/opentelemetry-core-mini/README.md

```diff
@@ -0,0 +1,3 @@
+This holds a very small subset of the `@opentelemetry/core` package, instead of
```

Comment:
See https://github.com/elastic/apm-agent-nodejs/pull/2641#issuecomment-1124295928 below.

> That said -- the two files we're manually vendoring haven't been edited since October and June of 2020 so if there is risk it seems low.

Yes, partly I'm banking on that, and also on tests to point out behaviour changes/incompatibilities.